tests/time_tests/test_runner/conftest.py

```diff
@@ -96,3 +105,42 @@ def get_dict_values(d):
     values = list(get_dict_values(val))
 
     return "-".join(["_".join([key, val]) for key, val in zip(keys, values)])
+
+
+class TestConfDumper:
```

Comment:
think about dumping references from DB to a file or something like that (DB data should be compatible with file with references for simple generation of references from DB)

Reply:
@asomsiko it's hard to convert data requested from DB to a new test config because current implementation supports environment variables which will be expanded in runtime and posted like this. 

We may post original instance values as `_orig` field for fast search and generation

Reply:
Or we may shift env vars substitution on a user that he do it manually after getting test config from DB with expanded env vars

Reply:
@vurusovs can we report/submit path before env expansion? There are multiple security [attacks ](https://cwe.mitre.org/data/definitions/22.html) mounted through path disclosure. I'd avoid expansion for that reason also

Reply:
OK, let's post data as is, expand variables only for execution

Reply:
Fixed